rc/core/comment.kak

```diff
@@ -78,7 +83,8 @@ hook global BufSetOption filetype=(pug|rust) %{
 }
 
 hook global BufSetOption filetype=python %{
-    set buffer comment_block '\'\'\':\'\'\''
+    set buffer comment_line '#'
+    set buffer comment_block "''':'''"
```

Comment:
The `comment_line` is already set to `#` by default, so we don't need to specify it here once again.